packages/deploy-tooling/src/cli/index.ts

```diff
@@ -60,6 +60,7 @@ export function createCommand(name: 'deploy' | 'undeploy'): Command {
         command
             .option('--package <abap-package>', 'Package name for deploy target ABAP system')
             .option('--description <description>', 'Project description of the app')
+            .option('--create-transport <description>', 'Create transport request during deployment')
```

Comment:
Dont forget to update the readme with this param too.

Reply:
Yep just saw this, pushing update

Reply:
`--create-transport` should have a conflict added with `--transport` as both are mutually exclusive.

Reply:
Updated